src/Http/Controllers/VoyagerBreadController.php

```diff
@@ -225,7 +225,7 @@ public function update(Request $request, $id)
         $this->authorize('edit', $data);
 
         // Validate fields with ajax
-        $val = $this->validateBread($request->all(), $dataType->editRows);
+        $val = $this->validateBread($request->all(), $dataType->editRows, __FUNCTION__);
```

Comment:
Same as the comment on the validateBread method. The validation per action is not stated on the docs. A PR of a bugfix should not contain new features.